pkg/reconciler/common/releases_test.go

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright 2020 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package common
+
+import (
+	"os"
+	"testing"
+
+	"github.com/tektoncd/operator/pkg/apis/operator/v1alpha1"
+	util "github.com/tektoncd/operator/pkg/reconciler/common/testing"
+)
+
+const (
+	VERSION          = "0.15.2"
+	TEKTON_PIPELINES = "testdata/kodata/tekton-pipeline/" + VERSION + "/release.notags.yaml"
```

Comment:
Can we remove this as i don't see any usage

Reply:
yup :+1: